tests/ignite/distributed/test_auto.py

```diff
@@ -80,8 +91,10 @@ def test_auto_methods_no_dist():
 def test_auto_methods_gloo(distributed_context_single_node_gloo):
 
     ws = distributed_context_single_node_gloo["world_size"]
-    _test_auto_dataloader(ws=ws, nproc=ws)
-    _test_auto_dataloader(ws=ws, nproc=ws, sampler_name="WeightedRandomSampler")
+    _test_auto_dataloader(ws=ws, nproc=ws, batch_size=1)
+    _test_auto_dataloader(ws=ws, nproc=ws, batch_size=10, num_workers=10)
+    _test_auto_dataloader(ws=ws, nproc=ws, batch_size=1, sampler_name="WeightedRandomSampler")
```

Comment:
Maybe we can remove this test ?

Reply:
I have removed it because I have no arguments for keeping it